config/locales/pl.yml

```diff
@@ -83,4 +84,5 @@ pl:
       new_event: 'Nowe wydarzenie'
     show:
       users_of_events: 'Lista osób biorących udział w wydarzeniu:'
-  
+    status: 
+      :incomming: 'Nadchodzący'
```

Comment:
- You cannot have `:` in front and in the end. 
- This translation is not in English version.